src/main/java/com/mide/gangsaeng/bannedword/BannedWord.java

```diff
@@ -0,0 +1,38 @@
+package com.mide.gangsaeng.bannedword;
+
+import java.util.Objects;
+
+import lombok.Getter;
+
+@Getter
+public class BannedWord {
+    private final String word;
+
+    public BannedWord(String word) {
+        validate(word);
+        this.word = word;
+    }
+
+    private void validate(String word) {
+        if (word == null) {
+            throw new IllegalArgumentException("null parameter is not allowed in banned word");
+        }
+
+        if (word.isBlank() || word.isEmpty()) {
```

Comment:
What's the difference between `isBlank` and `isEmpty` ?

Reply:
`isBlank()` returns true when the `String` doesn’t contain any character and all of its characters are whitespace characters. On the other hand, `isEmpty()` only returns true when the `String` doesn’t contain any character.

Reply:
Then why we need to call `isEmpty` after `isBlank`?

Reply:
I wanted to prevent empty string in `word`. In this case, it requires only `null` check and `isBlank()`.
 So i changed it.